tutorials/08-implementing-a-custom-feature.md

```diff
@@ -3,17 +3,17 @@
 ## Prerequisites
 
 - \ref installation "Installation"
-- \ref physicsplugin "Understand physics plugin"
-- \ref loadplugin "Load physics plugin"
-- \ref implementfeature "Implement physics feature"
+- \ref physicsplugin "Understanding the physics plugin"
+- \ref pluginloading "Loading physics plugins"
```

Comment:
Same here about changing the refs.

Reply:
Same as above - the current refs are broken, and these are fixes.